src/context.rs

```diff
@@ -2131,27 +2134,28 @@ impl ContextWriter {
 
     /* The left hand of two vertical rectangles always has a top right (as the
      * block above will have been decoded) */
-    let blk = &self.bc.at(bo);
-    if (blk.n4_w < blk.n4_h) && !blk.is_sec_rect {
+    if (target_n4_w < target_n4_h) && !is_sec_rect {
       has_tr = true;
     }
 
     /* The bottom of two horizontal rectangles never has a top right (as the block
      * to the right won't have been decoded) */
-    if (blk.n4_w > blk.n4_h) && blk.is_sec_rect {
+    if (target_n4_w > target_n4_h) && is_sec_rect {
       has_tr = false;
     }
 
     /* The bottom left square of a Vertical A (in the old format) does
      * not have a top right as it is decoded before the right hand
      * rectangle of the partition */
+/*
```

Comment:
Why disable this? Isn't it required to exclude some edge cases?

Reply:
I commented it out for now to avoid relying on blk (current block). Down the road we'll have to decide whether to write things into blk before calling find_mvrefs() or pass values as parameters. For now I am passing values as parameters.